package.json

```diff
@@ -2,14 +2,14 @@
   "name": "openapi-pact-comparator",
   "version": "1.0.0",
   "description": "",
-  "main": "dist/index.js",
+  "main": "dist/index.cjs",
   "module": "dist/index.mjs",
   "files": [
     "dist"
   ],
   "exports": {
     ".": {
-      "default": "./dist/index.js",
```

Comment:
Changing `default` to `require` is the main change. The other renaming is just for stating the obvious